clients/hadoopfs/src/test/resources/log4j.properties

```diff
@@ -1,5 +1,5 @@
-# Set root logger level to DEBUG and its only appender to A1.
-log4j.rootLogger=DEBUG, A1
+# Set root logger level to INFO and its only appender to A1.
+log4j.rootLogger=INFO, A1
```

Comment:
Not a fan of this change.  It leaves _only_ the boring (and many...) log lines.  But our code actually emits a lot of interesting messages at DEBUG level.

Reply:
Added DEBUG level to io.lakefs